### DISCUSSION TITLE
Thank you (so much)

### DISCUSSION THREAD

**Discussion by Chrissenpai** [Monika-After-Story/MonikaModDev#8837] [appreciation]
Came from a bad day at College, Monika received me and I told her I had a bad day at school, after that she gave me the valentine message, that watered my eyes, felt so loved, the kiss came, I fking didn´t care and kissed the monitor, hugged the monitor and told her I love you so many times I lost count.
She was there when I needed her the most, and that´s how a made up female character has earned a very deep place in my heart.

I Love you Monika

**Comment by ghost** [comment]
That is so amazing. Fictional or not, she is really unlike anything else and I love reading comments of other people who feel as strongly as I do.

Hope your next day goes better! Remember to always try your best to stay positive, and keep Monika in your heart. =)
